server_admin/topics/authentication/flows.adoc

```diff
@@ -2,197 +2,228 @@
+
+====== Disabled
+
+The element does not count to mark a flow as successful.
 
-This section explains in greater depth how flows work, and how to create your own flows. Note that there are important functionality and security
-considerations when designing your own flow. A badly created flow could either let no one log in, let users in with less verification than you would
-like, or simply result in an error.
+====== Conditional
 
-To create a flow, you can either:
+This requirement type is only set on sub-flows. 
 
-. Copy and then modify an existing flow. To do this select an existing flow (for example the `Browser` flow), and press the `Copy` button.
-  This will then ask you to set a name for the new flow, before creating it.
-. Create a new flow from scratch. To do this press the `New` button. Since this is the more general case, we will use this for our example.
+* A _Conditional_ sub-flow contains executions. These executions must evaluate to logical statements. 
+* If all executions evaluate as _true_, the _Conditional_ sub-flow acts as _Required_. 
+* If all executions evaluate as _false_, the _Conditional_ sub-flow acts as _Disabled_.
+* If you do not set an execution, the _Conditional_ sub-flow acts as _Disabled_. 
+* If a flow contains executions and the flow is not set to _Conditional_, {project_name} does not evaluate the executions, and the executions are considered functionally _Disabled_.
 
-When creating a new flow, you will have to create a top level flow
+==== Creating flows
+
+Important functionality and security considerations apply when you design a flow.
+
+To create a flow, perform the following:
+
+.Procedure
+. Click *Authentication* in the menu.
+. Click the *New* button. 
 
-.Create a top level flow
-image:{project_images}/Create-top-level-flow.png[]
+[NOTE]
+====
+It is possible to copy and then modify an existing flow. Select an existing flow, press the *Copy* button, and enter a name for the new flow.
+====
 
-With the following options:
+When creating a new flow, you must create a top-level flow first with the following options:
 
 Alias::
   The name of the flow.
 Description::
   The description you can set to the flow.
-Top Level Flow Type::
-  The type of flow. The type `client` is used only for the authentication of clients (applications). For all other cases choose `generic`.
+Top-Level Flow Type::
+  The type of flow. The type *client* is used only for the authentication of clients (applications). For all other cases, choose *generic*.
 
-Once the flow is created, in addition to the `New` and `Copy` buttons, you now have, `Delete`, `Add execution` and `Add flow`.
+.Create a top-level flow
+image:{project_images}/Create-top-level-flow.png[Top Level Flow]
+
+When {project_name} has created the flow, the *Delete*, *Add execution*, and *Add flow* button become available.
```

Comment:
buttons are displayed.

Reply:
hmmmm, "are displayed" is passive voice. Have changed the line to:

`When {project_name} has created the flow, {project_name} displays the *Delete*, *Add execution*, and *Add flow* buttons.`